docs/python-interface.md

```diff
@@ -25,13 +25,6 @@ To install it use the standard pip install commands:
 $ pip install spglib
 ```
 
-Release candidates can be found at the equivalent [PyPI testing project page](https://pypi.org/project/spglib/).
-You can test a specific release candidate locally using the following command:
-
-```console
-$ pip install -i https://test.pypi.org/simple/ spglib==2.1.0rc1
```

Comment:
How about informing users `pip install --pre spglib`?

Reply:
I have been debating that, there's the slight nuance mentioned in the description, but if they mention the version with `==` anyway, they wouldn't need it. Not sure how to word the instruction around it, or if there's an upstream documentation that would better cover us